vmdb/app/controllers/miq_request_controller.rb

```diff
@@ -141,8 +141,9 @@ def show_list
       gv_options = {:filter=>prov_condition(@sb[:def_prov_options][resource_type.to_sym])}
       @view, @pages = get_view(kls, gv_options) # Get view and paginator, based on the selected options
     else
-      requester = User.find_by_userid(session[:userid])
-      gv_options = {:filter=>prov_condition({:resource_type=>resource_type, :time_period=>time_period, :requester_id=>requester ? requester.id : nil})}
+      gv_options = {:filter => prov_condition(:resource_type => resource_type,
+                                              :time_period   => time_period,
+                                              :requester_id  => current_user.try(:id))}
```

Comment:
Would it be better to have a method `current_user_id` that does the try under the covers, so you don't have to repeat it?